tests/run.sh

```diff
@@ -1,5 +1,26 @@
 EXIT_STATUS=0
 
+# check the ulist versions first
+check_version() {
```

Comment:
@yingmanwumen Would you mind move this function into a separate file named `check_version.sh`. The `run.sh` file does not contain any function implementation and just call some existing files and commands.

Reply:
Hello, the separation is done ^_^